lms/lmsdb/bootstrap.py

```diff
@@ -231,22 +228,39 @@ def _add_api_keys_to_users_table(table: Model, _column: Field) -> None:
             user.save()
 
 
+def _add_uuid_to_users_table(table: Model, _colum: Field) -> None:
+    log.info('Adding UUIDs for all users, might take some extra time...')
+    with db_config.database.transaction():
+        for user in table:
+            user.uuid = uuid4()
+            user.save()
```

Comment:
Can't we save at the end of the transaction? I think it would take significantly less time